assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -24,7 +30,6 @@ isolated_cores={{.IsolatedCpus}}
 {{end}}
 
 not_isolated_cores_expanded=${f:cpulist_invert:${isolated_cores_expanded}}
-automatic_pstate=${f:intel_recommended_pstate}
```

Comment:
This is where you lost the intel_pstate=disable, it needs to go to the intel specific tuning profile